extensions/wikia/AdEngine/js/run/mercury.run.js

```diff
@@ -64,6 +66,10 @@ require([
 		if (adContext.getContext().opts.yavli) {
 			yavliTag.add();
 		}
+		
+		if (adInfoTracker && adInfoTracker.run) { //TODO: remove if check in 2017
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)